src/index.ts

```diff
@@ -1529,6 +1535,44 @@ function determineDragOperation(effectAllowed:string, dropEffect:string):string
     return DROP_EFFECTS[DROP_EFFECT.NONE];
 }
 
+function onDelayTouchstart(evt: TouchEvent) {
+    const el = evt.target;
+
+    const heldItem = () => {
+        end.off();
+        cancel.off();
+        scroll.off();
+        onTouchstart(evt);
+    };
+
+    const onReleasedItem = () => {
+        end.off();
+        cancel.off();
+        scroll.off();
+        clearTimeout(timer);
+    };
+
+    const timer = setTimeout(heldItem, config.holdToDrag);
+
+    const end = onEvt(el, 'touchend', onReleasedItem, this);
+    const cancel = onEvt(el, 'touchcancel', onReleasedItem, this);
+    const scroll = onEvt(window, 'scroll', onReleasedItem, this);
+}
+
+function onEvt(el: EventTarget, event: string, handler: () => any, context: any) {
+    if (context) {
+        handler = handler.bind(context);
+    }
+
+    el.addEventListener(event, handler, supportsPassive ? { passive: false } : false);
+
+    return {
+        off() {
+            return el.removeEventListener(event, handler, supportsPassive ? { passive: false } : false);
```

Comment:
This won't compile with TypeScript 2.3.3 as the signature for removeEventListener is not exactly the same as for addEventListener - maybe it's a bug in lib.d.ts.

Change to ` return el.removeEventListener(event, handler);` will make it compile. You can check by running `grunt` in your local checkout.

Reply:
I got the error locally yeah, fixed :)